src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with logical operators (&&, ||) or ternary operator (?:)
-  // without nesting
+  const men = people
+    .filter(item =>
```

Comment:
do not break line just after `=>`
it is not convenient, hard to read and I think may cause bugs during interpretation, like 
```
if (true) console.log('a')
console.log('b');
```
but I am not shure

Reply:
In this case we have only conditions, then I want indent them at the same column
I tried another indent. you will see